repo/packages/C/crate/0/config.json

```diff
@@ -0,0 +1,155 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+
+    "crate": {
+      "description": "Crate configuration properties.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "description": "The version number of Crate that should be used for the framework."
+        },
+        "cluster-name": {
+          "type": "string",
+          "description": "The name of the Crate cluster.",
+          "default": "crate"
+        },
+        "http-port": {
+          "type": "integer",
+          "description": "HTTP port of the Crate instances.",
+          "default": 4200,
+          "minimum": 0,
+          "maximum": 65535
+        },
+        "transport-port": {
+          "type": "integer",
+          "description": "Internal transport port of the Crate instances.",
+          "default": 4300,
+          "minimum": 0,
+          "maximum": 65535
+        },
+        "data-path": {
+          "type": "string",
+          "description": "The absolute path where Crate should store table data. Note that the user `crate` needs write access!"
+        },
+        "blob-path": {
+          "type": "string",
+          "description": "The absolute path where Crate should store blobs. Note that the user `crate` needs write access!"
+        },
+
+        "framework": {
+          "description": "Framework configuration properties.",
+          "type": "object",
+          "properties": {
+            "zookeeper": {
+              "default": "master.mesos:2181",
+              "type": "string",
+              "description": "URL to the Zookeeper Crate will communicate to (without protocol prefix, e.g. mesos-master:2181)"
+            },
+            "framework-name": {
```

Comment:
unfortunately this doesn't work.   there are a copy of details that need to be documented better on our end... the `framework-name` must be a property under `crate` and crate must be the same name that is in the package.json file (which is the case).    The health check on the dcos connection is made with these details.   To have this work we need:

1. the framework name that the scheduler registered with to be the value of the passed in framework-name.  This is a change at the framework level (you currently have it hard coded for crate-mesos)
2. We need a property for `crate.framework-name` with a default value of crate